emarket-traning-proj.iml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please, add `emarket-traning-proj.iml` to .gitignore

Reply:
Added `emarket-traning-proj.iml` to .gitignore.